profile/README.md

```diff
@@ -1 +1,32 @@
-## Hi there 👋
+![image](https://user-images.githubusercontent.com/67703407/135768793-fcd008f9-a9f6-4cfc-8230-7bb60143fafe.png)
+
+<h3 style="fontsize:30px" align='center'>Build and learn logic circuits in the cloud✨</h3>
+  
+<p align='center'>  
+  <a href="https://circuitverse.org/">CircuitVerse</a> is an easy to use <strong>digital logic circuit simulator </strong> which aims at providing a platform to create, share and learn digital circuits. It can run on almost any device without the need for installing any software. The platform has been designed for use by students, professionals and hobbyists alike. The vision is to grow a community around the platform that will aid students to self-learn digital logic design. Apart from the simulator, <strong>users can create, learn, collaborate and share their work <strong/>. CircuitVerse is currently used by several universities worldwide since it provides features for teachers to create groups and host assignments on the platform. The platform’s impact has been more evident than ever in the Covid 19 pandemic as CircuitVerse enabled schools and colleges to move their courses online.
+</p>
+
+ 
+---
+
+<p align="center">Get in touch</p>
+<p align="center"> 
+<a href="mailto:"><img alt="email_logo" 
+src="https://github.com/Aniket762/.github/blob/main/assets/email.svg"></a>
```

Comment:
please change this and other file src's below, from your own account to relative local paths of svgs of this repository